### ISSUE TITLE
Zellij: 

### ISSUE THREAD

**Issue by nelsonic** [dwyl/dev-setup#69]
Saw [**`@dhh`**](https://github.com/dwyl/learn-devops/issues/96#issuecomment-2411895687) and other Devs using `Zellij` recently and was intrigued. 💭 

https://zellij.dev
![image](https://github.com/user-attachments/assets/8e2ffe99-11b5-487f-ae81-e2715fb335f3)

https://github.com/zellij-org/zellij
![image](https://github.com/user-attachments/assets/c8ede6bb-d7e9-4def-9b25-b41c12dc90e3)

Turn Your Terminal Into A Tiling Window Manager With Zellij: https://youtu.be/Kc6ZuIIG3i8
<img width="1040" alt="image" src="https://github.com/user-attachments/assets/5022340b-14e7-46ea-86a3-26e1de5b25c5">

Going to give this a `try`. 
Currently using `warp` as my "personal" terminal https://github.com/dwyl/technology-stack/issues/123 but don't like the constant updates and nags at the top of the screen. And also don't _need_ my terminal to be "Ai Supercharged" (i.e. phoning home ...)

**Comment by ndrean** [comment]
Had a quick read, and trial. It's for VIM lovers, full of ctrl-XYZ. So unless you do heavy task swapping, my feeling is that this is something you want to be taught, and not  learn.

**Comment by nelsonic** [comment]
Yeah, got the same feeling. Aimed at `Vim`/`NeoVim` aficionados. 🤓 
We have a few in the @dwyl community. 🦄

**Comment by asntc** [comment]
Diving into this now! 🏊
